src/alloc.rs

```diff
@@ -309,15 +309,19 @@ enum Block {
     Entry(core::num::NonZeroUsize, usize, bool),
 }
 
-/// The metadata for each ptr is only 2 machine words which means that copying
-/// the data out of the `AllocList` is not that expensive. This is done so that
-/// the API is cleaner:
-///     * We abstract away the complexities of the underlying block structure
-///     * It is easier to work with primitives than references to primitives
+/// Information about an allocation block used by the collector.
+///
+/// This information is stored for every single allocation. It is vital for the
+/// collector to know things such as: determining whether an arbitrary word in a
+/// program is a ptr; how to get to the beginning of a block from some inner
+/// ptr; and whether a block is managed by the collector.
 #[derive(Debug, PartialEq, Eq, Copy, Clone)]
 pub struct PtrInfo {
+    /// The pointer to the beginning of the allocation block.
     pub ptr: usize,
+    /// The size of the allocation block.
```

Comment:
In bytes, I assume?

Reply:
Yes. Fixed: ac8201e1afce6cb544f5cc6fab81324b076c4ba0